build/android/build.gradle

```diff
@@ -54,3 +61,62 @@ android {
 		}
 	}
 }
+
+task cleanAssets(type:Delete) {
+	delete 'src/main/assets'
+}
+
+task cleanIconv(type:Delete) {
+	delete 'deps/libiconv'
+}
+
+task cleanIrrlicht(type:Delete) {
+	delete 'deps/irrlicht'
+}
+
+task cleanLevelDB(type:Delete) {
+	delete 'deps/leveldb'
+}
+
+task cleanCURL(type:Delete) {
+	delete 'deps/curl'
+	delete 'deps/curl-' + curl_version
+}
+
+task cleanOpenSSL(type:Delete) {
+	delete 'deps/openssl'
+	delete 'deps/openssl-' + openssl_version
+	delete 'deps/openssl-' + openssl_version + '.tar.gz'
+}
+
+task cleanOpenAL(type:Delete) {
+	delete 'deps/openal-soft'
+}
+
+task cleanFreetype(type:Delete) {
+	delete 'deps/freetype2-android'
+}
+
+task cleanOgg(type:Delete) {
+	delete 'deps/libvorbis-libogg-android'
+}
+
+task cleanSQLite3(type:Delete) {
+	delete 'deps/sqlite-amalgamation-' + sqlite3_version
+	delete 'deps/sqlite-amalgamation-' + sqlite3_version + '.zip'
+}
+
+task cleanGMP(type:Delete) {
+	delete 'deps/gmp'
+	delete 'deps/gmp-' + gmp_version
+}
+
+task cleanAll(type:Delete, dependsOn:[clean,cleanAssets,cleanIconv,cleanFreetype,cleanIrrlicht,cleanLevelDB,cleanSQLite3,cleanCURL,cleanOpenSSL,cleanOpenAL,cleanOgg,cleanGMP]) {
```

Comment:
Line is too long

Reply:
it's gradle, do we got same problem ? :p